src/Microsoft.AspNet.Localization/CookieRequestCultureProvider.cs

```diff
@@ -17,6 +17,10 @@ public class CookieRequestCultureProvider : RequestCultureProvider
         private static readonly char[] _cookieSeparator = new[] { '|' };
         private static readonly string _culturePrefix = "c=";
         private static readonly string _uiCulturePrefix = "uic=";
+        /// <summary>
```

Comment:
Do you mean after ```_uiCulturePrefix```? I already added what I suggest with the docs, i'm not sure if empty line is needed